lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -242,7 +250,11 @@ export abstract class ClientApplication {
                         this.eventHandler.emitEvent(EventType.LOGIN_FAILURE, InteractionType.Redirect, null, e);
                     }
                     this.eventHandler.emitEvent(EventType.HANDLE_REDIRECT_END, InteractionType.Redirect);
-
+                    handleRedirectPromiseMeasurement?.endMeasurement({
+                        errorCode: e instanceof AuthError && e.errorCode || undefined,
```

Comment:
This would translate to e.errorCode || undefined. [For more detail on logical operators](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_AND#description)